internal/internal_logging_tags.go

```diff
@@ -21,25 +21,26 @@
 package internal
 
 const (
-	tagActivityID        = "ActivityID"
-	tagActivityType      = "ActivityType"
-	tagDomain            = "Domain"
-	tagEventID           = "EventID"
-	tagEventType         = "EventType"
-	tagRunID             = "RunID"
-	tagTaskList          = "TaskList"
-	tagTimerID           = "TimerID"
-	tagWorkflowID        = "WorkflowID"
-	tagWorkflowType      = "WorkflowType"
-	tagWorkerID          = "WorkerID"
-	tagWorkerType        = "WorkerType"
-	tagSideEffectID      = "SideEffectID"
-	tagChildWorkflowID   = "ChildWorkflowID"
-	tagLocalActivityType = "LocalActivityType"
-	tagLocalActivityID   = "LocalActivityID"
-	tagQueryType         = "QueryType"
-	tagVisibilityQuery   = "VisibilityQuery"
-	tagPanicError        = "PanicError"
-	tagPanicStack        = "PanicStack"
-	causeTag             = "pollerrorcause"
+	tagActivityID            = "ActivityID"
+	tagActivityType          = "ActivityType"
+	tagDomain                = "Domain"
+	tagEventID               = "EventID"
+	tagEventType             = "EventType"
+	tagRunID                 = "RunID"
+	tagTaskList              = "TaskList"
+	tagTimerID               = "TimerID"
+	tagWorkflowID            = "WorkflowID"
+	tagWorkflowType          = "WorkflowType"
+	tagWorkerID              = "WorkerID"
+	tagWorkerType            = "WorkerType"
+	tagSideEffectID          = "SideEffectID"
+	tagChildWorkflowID       = "ChildWorkflowID"
+	tagLocalActivityType     = "LocalActivityType"
+	tagLocalActivityID       = "LocalActivityID"
+	tagQueryType             = "QueryType"
+	tagVisibilityQuery       = "VisibilityQuery"
+	tagPanicError            = "PanicError"
+	tagPanicStack            = "PanicStack"
+	causeTag                 = "pollerrorcause"
+	tagworkflowruntimelength = "workflowruntimelength"
```

Comment:
all the tags above have PascalCase naming convention except `pollererrorcause`. better to keep these consistent. at least for the new one you are introducing

Reply:
Changing an existing tag might cause problems in metric dashboards that refer to it. Might worth double checking dependencies before improving the name to fit the convention